app/assets/stylesheets/custom.css.scss

```diff
@@ -133,6 +133,13 @@ a:hover, a:active {
     margin-bottom: 20px;
 }
 
+.btn-details {
+    background-color: $white;
+    border-color: $clam-tan;
+    margin-top: 10px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background-color, border-color, color, margin, margin-top